packages/web-components/fast-components/src/radio/radio.ts

```diff
@@ -91,6 +91,14 @@ export class Radio extends FormAssociated<HTMLInputElement> {
 
     public connectedCallback(): void {
         super.connectedCallback();
+        if (
```

Comment:
I'm wondering if this ties the radio too tightly to the radio-group. What if someone wanted to use a *different* mechanism to manage focus on the radio?

Is there a way to just make the control react correctly to change to the tabindex attribute?

Reply:
This seems problematic because the connectedCallback or if something was in the template for radio happens after connectedCallback in the radio-group, thus overriding anything set by radio-group or any other container presumably. I'm not sure it should be set at all in FASTRadio

Reply:
Okay lets leave this as-is then - we can always circle back if we need to and figure out a more robust solution.

Reply:
Thanks for investigating